test/test_conversion_driver_esm1p5.py

```diff
@@ -77,20 +77,18 @@ def test_find_matching_fields_files():
 
 def test_convert_fields_file_list_single():
     with mock.patch("um2netcdf4.process") as mock_process:
-        mock_process.return_value = None
```

Comment:
Setting the return value needs to be left in to ensure `None` is returned, otherwise the mock module returns another mock object, "representing" the function call:
```Python
>>> m1 = mock.Mock()
>>> res = m1.process()  # call a mock without setting anything on it
>>> res
<Mock name='mock.process()' id='126979898960448'>
>>>
>>> m1.process2.return_value = 5
>>> res2 = m1.process2()
>>> res2  # should return a value instead of a mock
5
```